### PR TITLE
Feature: User Permission Condition

### PR DESCRIPTION
**Pull request by madsrasmussen**

### CODE REVIEW

File: src/packages/documents/documents/entity-actions/culture-and-hostnames/manifests.ts [hunk 1]
```diff
@@ -16,6 +16,12 @@ const entityActions: Array<ManifestEntityAction> = [
 			repositoryAlias: UMB_DOCUMENT_DETAIL_REPOSITORY_ALIAS,
 			entityTypes: [UMB_DOCUMENT_ENTITY_TYPE],
 		},
+		conditions: [
+			{
+				alias: 'Umb.Condition.UserPermission',
+				verbs: ['Umb.Document.CultureAndHostnames'],
```
Comment (nielslyngsoe) [inline src/packages/documents/documents/entity-actions/culture-and-hostnames/manifests.ts:22]: typically for the other conditions we use 'match' or 'anyOf' for the matching kind of conditions.

Comment (nielslyngsoe) [reply src/packages/documents/documents/entity-actions/culture-and-hostnames/manifests.ts:22]: or allOf

File: src/packages/documents/documents/entity-actions/culture-and-hostnames/manifests.ts [hunk 2]
```diff
@@ -16,6 +16,12 @@ const entityActions: Array<ManifestEntityAction> = [
 			repositoryAlias: UMB_DOCUMENT_DETAIL_REPOSITORY_ALIAS,
 			entityTypes: [UMB_DOCUMENT_ENTITY_TYPE],
 		},
+		conditions: [
+			{
+				alias: 'Umb.Condition.UserPermission',
```
Comment (nielslyngsoe) [inline src/packages/documents/documents/entity-actions/culture-and-hostnames/manifests.ts:21]: then maybe this one should be called something with Verbs?..

Comment (nielslyngsoe) [reply src/packages/documents/documents/entity-actions/culture-and-hostnames/manifests.ts:21]: "Umb.Condition.DocumentUserPermission"

**Comment by nielslyngsoe** [comment]
@madsrasmussen notice there is now a UmbConditionBase class that these conditions can extend. :-)

**Comment by madsrasmussen** [comment]
This PR will be replaced by a new one. I will close this.

**Pull request closed without merging**